Modules/EMCAL/src/DigitsQcTask.cxx

```diff
@@ -14,6 +14,7 @@
 #include "EMCAL/DigitsQcTask.h"
 #include "DataFormatsEMCAL/Cell.h"
 #include "EMCALBase/Geometry.h"
+#include "Framework/InputRecord.h"
```

Comment:
```suggestion
#include <Framework/InputRecord.h>
```
Perhaps <> braces to let the compiler find the header faster, as it is should look for it in other project.

Reply:
absolutely, fixing all of htem